### DISCUSSION TITLE
Charging Status on POE

### DISCUSSION THREAD

**Discussion by jsnyder81** [WLAN-Pi/feedback#14] [Hardware - WLAN Pi Pro] [answered]
just got home and am testing the WLANPi Pro.  Charging via POE and the status page shows discharging, while the PoE light is on and the percentage is going up.

![DB70FCF9-6915-4BAA-B864-96DFE0AD0687](https://user-images.githubusercontent.com/5827846/155861728-907bdde6-f68e-4fe3-bb79-4ef8e08ec5b6.jpeg)

**Comment by jolla** [comment]
That would indicate that the WLAN Pi Pro is consuming more energy than is being supplied. Could you check how much power your PSE is delivering? It should be 12W (2A).

Is anything connected to the WLAN Pi Pro USB ports that would drawing power?

**Comment by jsnyder81** [comment]
The pi pro is powering an original wlan pi via USB.  However, the battery percentage went from 80% to 100% all while saying “discharging.”

I show it consuming 9.2W currently and it’s topped to 100%.

**Comment by jolla** [comment, marked as answer]
Ok, it looks like there is an issue where if more power is being drawn then expected the charging controller misreports as discharging even though it does appear to still be charging. We'll need to investigate this further to see if it can be addressed in a firmware update with the micro controller.
